src/App.tsx

```diff
@@ -1,24 +1,233 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { Loader } from './components/Loader';
+import { FilterTypes } from './components/TodoFilter';
+import { Todos } from './components/Todos';
+import { Todo } from './types/Todo';
+import { Footer } from './components/Footer';
+import {
+  deleteTodos,
+  getTodos,
+  patchTodos,
+  postTodos,
+} from './api/todos';
+import { TodoErrors } from './types/Errors';
+import { Header } from './components/Header/Header';
 
-const USER_ID = 0;
+export const USER_ID = '10682';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [input, setInput] = useState<string>('');
+  const [tempTodoId, setTempTodoId] = useState<number | null>(null);
+  const [filter, setFilter] = useState(FilterTypes.All);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [error, setError] = useState<string>('');
```

Comment:
```suggestion
  const [error, setError] = useState<TodoErrors>(TodoErrors.None);
```

Reply:
would be better to do it this way. Just add `None` option to `TodoErrors` with `''` value